app/services/numbers.js

```diff
@@ -0,0 +1,17 @@
+const request = require('../helpers/request');
+const {
+  common: { numbersApi }
+} = require('../../config');
+const { SERVER_ERROR } = require('../constants/errors');
+const logger = require('../logger');
+
+exports.getSentence = async () => {
+  const { url } = numbersApi;
+  try {
+    const data = await request.get(url);
+    return data.data;
+  } catch (err) {
+    logger.error(err);
+    throw Error(SERVER_ERROR);
```

Comment:
You should throw one our errors, not the Error object of JS. I think it will not even work without the `new` keyword.
With 'our errors' I mean the errors listed on errors file at app folder.